ui/package.json

```diff
@@ -23,22 +23,24 @@
   },
   "dependencies": {
     "@babel/preset-typescript": "^7.12.7",
-    "@material-ui/core": "4.11.1",
+    "@material-ui/core": "^4.11.3",
     "@material-ui/icons": "^4.0.0",
     "@material-ui/lab": "^4.0.0-alpha.56",
     "@material-ui/styles": "^4.11.3",
     "leaflet": "^1.0.2",
     "leaflet.markercluster": "^1.4.1",
     "material-ui-dropzone": "^3.5.0",
-    "mui-virtualized-table": "3.0.0-6",
+    "mui-virtualized-table": "^2.2.3",
     "prop-types": "^15.7.2",
-    "react": ">=15",
-    "react-dom": ">= 16.8",
+    "react": "^15.3.0 || ^16.0.0-alpha",
+    "react-dom": "^15.3.0 || ^16.0.0-alpha",
     "react-is": "^16.13.1",
     "react-leaflet": "^2.0.0",
     "react-leaflet-control": "^2.1.2",
     "react-leaflet-markercluster": "^2.0.0",
     "react-router-dom": "^5.2.0",
+    "react-virtualized": "^9.22.3",
```

Comment:
Lol kinda confused, I accidentally deleted react-window instead of virtualized but everything seems to work normally... will ask u about this on Saturday.